src/AppBundle/Resources/config/services.yml

```diff
@@ -1,4 +1,13 @@
 services:
+    ezplatform.pre_content_view_listener:
+        class: AppBundle\EventListener\PreContentViewListener
+        arguments:
+            - blogLocationId: '%blog.location_id%'
+        tags:
+            - name: kernel.event_listener
+              event: ezpublish.pre_content_view
+              method: onPreContentView
```

Comment:
I would prefer to keep like that because git-diff looks better in case of any further changes.